integrations/push-protocol/components/channel-card.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react'
+
+import CopyToClipboard from 'react-copy-to-clipboard'
+import { HiUser } from 'react-icons/hi'
+
+import { Loadable } from './loadable'
+import { SubscribeButton } from './subscribe-button'
+import { ChannelCardProps } from './types'
+import { useChannel } from '../hooks'
+import { strLimit, truncateAddress } from '../utils/helpers'
+
+export function ChannelCard(props: ChannelCardProps) {
+  const { env, channelAddress, onSubscribe, onUnsubscribe } = props
+  const {
+    data: channel,
+    isLoading: channelIsLoading,
+    error,
+  } = useChannel({
+    channel: channelAddress,
+    env: env,
+  })
+
+  const [copied, setCopied] = useState(false)
+
+  useEffect(() => {
+    if (!copied) return
+    setTimeout(() => setCopied(false), 3000)
+  }, [copied])
+
+  if (!channelAddress) return <>No Channel Address Specified...</>
+  if (error) return <>Error loading channel...</>
+
+  return (
+    <Loadable isLoading={channelIsLoading}>
+      {channel && (
+        <div className="flex w-full space-x-4">
+          <div className="flex flex-col">
+            <img alt={channel.name} className="w-24 md:w-32" src={channel.icon} />
```

Comment:
Rounded images look better
```suggestion
            <img alt={channel.name} className="w-24 rounded-xl md:w-32" src={channel.icon} />
```